docs/setup.md

```diff
@@ -100,3 +138,73 @@ git -C web/themes/contrib/atomic ls-remote --get-url
 # Step 5: Setup npm linked packages for theme dependencies
 npm run local:theme-link
 ```
+
+## Project Commands
+
+__NOTE: There are more commands, but these are those that current developers feel are most used.__
+
+### NPM
+
+```bash
+npm run setup                                   # Setup whole environment
+```
+```bash
+npm run build-with-assets                       # Build/Rebuild with replaced assets
+```
+```bash
+npm run build-with-install                      # Rebuild with fresh imports
+```
+```bash
+npm run local:cl-dev                            # npm link component library with local environment
+```
+```bash
+npm run local:review-with-atomic-and-cl-branch  # Use different branches of each repo
```

Comment:
I'm curious if we should get rid of these scripts / not talk about them because your `git-checkout` script is so good I'm inclined to say we can replace those others with yours. :)

Reply:
😁 That makes me so happy.  I'm happy for us to discuss this and we could talk about retiring them or archiving them?  Maybe we'll wait on that till Jim gets back so we can get his input.  I'd hate to remove any if someone is using it still and prefer it.

Reply:
+1 to remove scripts if they are not being used!

Reply:
Ok I removed some of them from the docs to instead attempt to use the git-checkout.

Reply:
+1 to remove scripts that aren't being used (outside the scope of this PR), to undo unnecessary nesting of scripts, and to be sure scripts have good comments. Since we're removing the details of what the scripts are doing on the README (which I think is a good idea) we'll want the scripts to be easy to decipher.